google-userinfo/background/main.js

```diff
@@ -0,0 +1,23 @@
+
+function notifyUser(user) {
+  browser.notifications.create({
+    "type": "basic",
+    "title": "Google info",
+    "message": `Hi ${user.name}`
+  });}
+
+function logError(error) {
+  console.error(`Error: ${error}`);
+}
+
+/*
+When the button's clicked:
+- get an access token using the identity API
+- use it to get the user's info
+- show a notification containing some of it
+*/
+browser.browserAction.onClicked.addListener(() => {
+  getAccessToken().then(getUserInfo)
+                  .then(notifyUser)
+                  .catch(logError);
```

Comment:
Like this:

```
browser.browserAction.onClicked.addListener(() => { 
   getAccessToken().then(getUserInfo)
   .then(notifyUser)
   .catch(logError);
```

It seems less readable to me, but if that's the rule, then OK. (I have no idea really how indentation should work with thens).